component/LinkableText/LinkableText.tsx

```diff
@@ -0,0 +1,25 @@
+import Link from "next/link";
+
+const LinkableText = ({text}: {text: string}) => {
+    const sanitizeNewLine = text?.split('\n')
+    const regexUrl = /^((?:https?:\/\/)?[^./]+(?:\.[^./]+)+(?:\/.*)?)$/
+
+    const textComponent = sanitizeNewLine?.map((textline, index) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sanitizeNewLine` variable is not null-checked before being used. This could lead to a runtime error if `text` is undefined or null. Consider adding a null check before using `text`.

```diff
-    const sanitizeNewLine = text?.split('\n')
+    const sanitizeNewLine = text ? text.split('\n') : []
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->